app/root/root.tsx

```diff
@@ -41,6 +41,7 @@ export default class RootComponent extends React.Component {
       next: (user: User) => this.setState({ ...this.state, user }),
     });
     faviconService.setDefaultFavicon();
+    window._preferences = this.state.preferences;
```

Comment:
intentional?

Reply:
Ah yes, I was using this to be able to quickly switch between dense and non-dense on a given page. Was calling `_preferences.toggleDenseMode()` in the console